### DISCUSSION TITLE
RevitAddInManager for Revit 2017

### DISCUSSION THREAD

**Discussion by AlexRubanMFT** [chuongmep/RevitAddInManager#46] [General]
Dear Revit users!

I found on this portal a link for installing the RevitAddInsManager only starting in 2018. For various reasons, I still use the Revit 2017.

If someone has saved the file for installing the  RevitAddInsManager 2017, then I will be very grateful if you share it with me.

Thank you very much in advance!

Best regards,
Alex Ruban

**Comment by chuongmep** [comment]
Hi @AlexRubanMFT you can resolve by this step : 
 - Install latest version [RevitAddinamanger ](https://github.com/chuongmep/RevitAddInManager/releases/latest)
 - Go to folder Addin version 2018, in this case example is path on my computer `C:\Users\vho2\AppData\Roaming\Autodesk\Revit\Addins\2018`
 - Copy Folder `RevitAddinManager` and file `RevitAddinManager.addin` to folder Addin version 2017
 - Open Revit and use it.
 Regard !

**Reply by AlexRubanMFT** [reply]
I did everything as you wrote. Everything seems to be working!

Thanks a lot!
